README.md

```diff
@@ -1,2 +1,44 @@
-# Restaurant-Page
-A simple restaurant page built using Vanilla Javascript, Bootstrap and CSS
+# Restaurant Page
+
+A simple restaurant page built using Vanilla Javascript to generate its entire content.
+
+![screenshot](screenshot.png)
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/Abidoyinsola1/Restaurant-Page/feature/dist/index.html)
+
+## Built With
+
+- Javascript
+- HTML & CSS3
+- Bootstrap
+
+## Prerequisites
+
+- Node
+- Get a browser like Chrome and Firefox in their most recent versions
+
+### Setup
+
+- Clone the repository on your local machine
+- cd into the folder
+- npm install
+- npm run build
+- Open the dist/index.html file in any web browser of your choosing.
```

Comment:
Try to be specific on how to switch directories with commands. make sure sure that each step, you put the command to do it